sympy/concrete/summations.py

```diff
@@ -253,6 +257,174 @@ def _eval_simplify(self, ratio, measure):
+        try:
+            if limit(sequence_term, sym, upper_limit) != S.Zero:
+                return False
+        except NotImplementedError:
+            pass
+
+        order = O(sequence_term, (sym, S.Infinity))
+
+        ### --------- p-series test (1/n**p) ---------- ###
+        p1_series_test = order.expr.match(sym**p)
+        if not p1_series_test is None:
+            if p1_series_test[p] < -1:
+                return True
+            if p1_series_test[p] > -1:
+                return False
+
+        p2_series_test = order.expr.match((1/sym)**p)
+        if not p2_series_test is None:
+            if p2_series_test[p] > 1:
+                return True
+            if p2_series_test[p] < 1:
+                return False
+
+        ### ----------- root test ---------------- ###
+        lim = Limit(abs(sequence_term)**(1/sym), sym, S.Infinity)
+        lim_evaluated = lim.doit()
+        if lim_evaluated.is_number:
+            if lim_evaluated < 1:
+                return True
+            if lim_evaluated > 1:
+                return False
+
+        ### ------------- alternating series test ----------- ###
+        d = symbols('d', cls=Dummy)
+        dict_val = sequence_term.match((-1)**p*q)
+        if dict_val[p] == sym or dict_val[p].has(sym):
+            if solveset((dict_val[q].subs({sym: d + 1})/dict_val[q].subs({sym: d})) < 1, d, interval) == interval and \
+                limit(dict_val[q], sym, S.Infinity).is_finite:
+                    return True
+            return False
+
+        ### ------------- comparison test ------------- ###
+        # (1/log(n)**p) comparison
+        log_test = order.expr.match(1/(log(sym)**p))
+        if not log_test is None:
+            return False
+
+        # (1/(n*log(n)**p)) comparison
+        log_n_test = order.expr.match(1/(sym*(log(sym))**p))
+        if not log_n_test is None:
+            if log_n_test[p] > 1:
+                return True
+            return False
+
+        # (1/(n*log(n)*log(log(n))*p)) comparison
+        log_log_n_test = order.expr.match(1/(sym*(log(sym)*log(log(sym))**p)))
+        if not log_log_n_test is None:
+            if log_log_n_test[p] > 1:
+                return True
+            return False
+
+        if order.expr == sequence_term:
+            pass
+        else:
+            return Sum(order.expr, (sym, lower_limit, upper_limit)).is_convergent()
+
+        ### ------------- integral test -------------- ###
+        if is_decreasing(sequence_term, interval):
+            integral_val = Integral(sequence_term, (sym, lower_limit, upper_limit))
+            try:
+                integral_val_evaluated = integral_val.doit()
+                if integral_val_evaluated.is_infinite:
+                    return False
+            except NotImplementedError:
+                pass
+            return True
+
+        ### -------------- Dirichlet tests -------------- ###
+        a_n = order.expr.args[0]
+        b_n = order.expr.args[1]
```

Comment:
I have the added the `.is_Mul` assumption checker. I am not sure if any more would be required.